### PR TITLE
Fix a potential flake in the tests for podman cp

### PR DESCRIPTION
**Pull request by mheon**
Instead of using the working directory, use a subdirectory of the temporary directory created for the individual test, to prevent a potential EEXIST for shared working directory.

### CODE REVIEW

**Comment by rhatdan** [comment]
TESTS are not happy, but the change LGTM

File: test/e2e/cp_test.go [hunk 1]
```diff
@@ -139,8 +130,5 @@ var _ = Describe("Podman cp", func() {
 		session = podmanTest.Podman([]string{"cp", name + ":/foo.tar.gz", "-"})
 		session.WaitWithDefaultTimeout()
 		Expect(session.ExitCode()).To(Equal(0))
```
Comment (TomSweeneyRedHat) [inline test/e2e/cp_test.go]: so the gz file gets removed via another mechanism?

**Comment by TomSweeneyRedHat** [comment]
LGTM and happy green test buttons.
One question, most likely informational for me.

**Pull request merged**